src/plone/app/mosaic/transform.py

```diff
@@ -171,16 +166,17 @@ def transformIterable(self, result, encoding):
                 else:
                     body_classes.append('layout-custom')
 
-        # Enable mosaic-grid when no grid system is defined
-        gridSystem = xpath1(gridXPath, result.tree)
-        if gridSystem is None:
-            registry = queryUtility(IRegistry)
-            if registry:
-                settings = registry.forInterface(IBlocksSettings, check=False)
-                gridSystem = getattr(
-                    settings, 'default_grid_system', None) or None
-        if gridSystem is None:
-            body_classes.append('mosaic-grid')
+        # # Enable mosaic-grid when no grid system is defined
+        # gridSystem = xpath1(gridXPath, result.tree)
+        # if gridSystem is None:
+        #     registry = queryUtility(IRegistry)
+        #     if registry:
+        #         settings = registry.forInterface(IBlocksSettings, check=False)
+        #         gridSystem = getattr(
+        #             settings, 'default_grid_system', None) or None
+        # if gridSystem is None:
+        #     body_classes.append('mosaic-grid')
+        body_classes.append('mosaic-grid')
```

Comment:
@agitator This should be unnecessary now and could also be removed (not only commented out).